CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # [develop](https://github.com/adhearsion/electric_slide)
+  * Added `ElectricSlide::CallQueue#update_agent` to safely update a queued agent object's attributes
```

Comment:
The way I remember it, the original intent was to use `#add_agent` whether you were adding a new agent, or updating an existing one. I also recall that at some point we added protection to prevent the same agent from being added twice, which may have broken that use-case. `#return_agent` isn't necessarily appropriate for updating agent details because its intent is to mark an agent available again. So it may be that we need an `#update_agent` method, I just wanted to think through it and stay as DRY as possible.